app/Services/IndexFacultiesService.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Services;
+
+use App\Models\Faculty;
+use Illuminate\Database\Eloquent\Collection;
+
+class IndexFacultiesService
+{
+    public function index(): Collection
+    {
+        return Faculty::all();
```

Comment:
Do you have all object witch full relations ? If not then load everything but without field relations. It will be much more efficient to build a query on a query bilder and more readable if you add a static function to the model where it can be reused.